h/views/admin/groups.py

```diff
@@ -59,50 +59,48 @@ def __init__(self, request):
 
     @view_config(request_method="GET")
     def get(self):
+
         self.form.set_appstruct(
             {
                 "creator": self.request.user.username,
                 "organization": Organization.default(self.request.db).pubid,
+                "enforce_scope": True,
             }
         )
         return self._template_context()
 
     @view_config(request_method="POST")
     def post(self):
         def on_success(appstruct):
+            """Create a group on successful validation of POSTed form data"""
+
             group_create_svc = self.request.find_service(name="group_create")
```

Comment:
One of the things I'd like to do in a subsequent refactor is better organize how the Controller classes deal with their services. Having them scattershot through the methods makes test configuration obnoxious and is less than ideal overall.

Reply:
We should also rename the classes to stop calling them "controller" classes, since Pyramid doesn't have controllers, they're called "view classes" in the Pyramid docs.

Reply:
Agreed. It's confusing and misleading.